resources/assets/sass/components/_gallery.scss

```diff
@@ -52,3 +54,21 @@
             height: auto;
             transform: translate(-50%, -50%);
         }
+
+        .c-gallery__more {
```

Comment:
Is this a real issue which should be resolved in scss or can it be fixed by changing hound settings?